state/remote/atlas.go

```diff
@@ -48,7 +48,7 @@ func atlasFactory(conf map[string]string) (Client, error) {
 
 	parts := strings.Split(name, "/")
 	if len(parts) != 2 {
-		return nil, fmt.Errorf("malformed name '%s'", name)
+		return nil, fmt.Errorf("malformed name '%s' (valid format is 'username/name')", name)
```

Comment:
Totally - I agree it is confusing. I think we are still exploring to find the best UX here, so any feedback is definitely appreciated! :smile: